en/tutorials/land_detector.md

```diff
@@ -4,45 +4,49 @@ The land detector is a dynamic vehicle model representing key vehicle states suc
 
 ## Configuring Auto-Disarming
 
-By default the land detector does detect landing, but does not auto-disarm. If the hysteresis parameter `COM_DISARM_LAND` is set to a non-zero value the system will auto-disarm after N seconds \(the value it is set to\).
+By default the land detector does detect landing, but does not auto-disarm. If the hysteresis parameter [COM_DISARM_LAND](../advanced/parameter_reference.md#COM_DISARM_LAND) is set to a non-zero value the system will auto-disarm after N seconds \(the value it is set to\).
 
 ## Multicopter Land Detector Configuration
 
-The complete set of parameters is available in the QGroundControl parameter editor under the `LNDMC` prefix. The key parameters that might differ per airframe are these:
+The complete set of parameters is available in the *QGroundControl* parameter editor under the `LNDMC` prefix. The key parameters that might differ per airframe are these:
 
-* `LNDMC_MAN_DWNTHR` - the threshold \(in percent, default is 15%\) for how much manual throttle is allowed to be considering the state as landed. Systems with very high thrust-to-weight ratios like racers might need a lower setting here \(e.g. 8%\).
```

Comment:
@julianoes 
1. LNDMC_MAN_DWNTHR was removed, which is fine, but how has the land detector changed correspondingly? Is the value now hard coded or is throttle no longer a relevant parameter (ie can I ignore the functionality this used to have. 
2. Is there anything else that needs to change in this doc, either because of this removal, or because you "just happen to know" there are other issues with it? (i.e. are there any other known changes in land detection we need to record).